### PR TITLE
[REF] base_automation: convert error dialog to wowl

### PR DESCRIPTION
**Pull request by sdegueldre**

### CODE REVIEW

File: addons/base_automation/static/src/xml/base_automation_error_dialog.xml [hunk 1]
```diff
@@ -1,36 +1,34 @@
...
+                as you are effectively disabling a customization that may set
+                important and/or required fields.
+            </p>
+            <p t-else="">
+                You can ask an administrator to disable or correct this automated action.
+            </p>
+        </xpath>
+        <xpath expr="//div[@role='alert']//button" position="after">
+            <t t-if="isUserAdmin">
+                <button class="btn btn-secondary mt4 o_disable_action_button" t-on-click="disableAction">
```
Comment (ged-odoo) [inline addons/base_automation/static/src/xml/base_automation_error_dialog.xml]: maybe if you want, you can do: `t-on-click.prevent="disableAction"`

File: addons/base_automation/static/src/xml/base_automation_error_dialog.xml [hunk 2]
```diff
@@ -1,36 +1,34 @@
...
                     <i class="fa fa-ban mr8"/>Disable Action
                 </button>
-                <button class="btn btn-secondary mt4 o_edit_action_button">
+                <button class="btn btn-secondary mt4 o_edit_action_button" t-on-click="editAction">
```
Comment (ged-odoo) [inline addons/base_automation/static/src/xml/base_automation_error_dialog.xml]: same here

File: addons/base_automation/static/src/js/base_automation_error_dialog.js [hunk 3]
```diff
@@ -1,78 +1,60 @@
...
+    editAction(ev) {
+        this.actionService.doAction({
+            name: "Automated Actions",
+            res_model: "base.automation",
+            res_id: this.actionId,
+            views: [[false, "form"]],
+            type: "ir.actions.act_window",
+            view_mode: "form",
+        });
+        this.close();
```
Comment (aab-odoo) [inline addons/base_automation/static/src/js/base_automation_error_dialog.js:54]: doesn't close automatically when executing another action?

**Review by aab-odoo — APPROVED** [review]

File: addons/base_automation/static/src/js/base_automation_error_dialog.js (hunk 3 above)
Comment (sdegueldre) [reply addons/base_automation/static/src/js/base_automation_error_dialog.js:54]: Just checked: it doesn't.

**Comment by aab-odoo** [comment]
robodoo rebase-ff r+

**Pull request closed without merging**